src/xtopcom/xrpc/xedge/xedge_method_manager.hpp

```diff
@@ -187,6 +193,23 @@ void xedge_method_base<T>::sendTransaction_method(xjson_proc_t & json_proc, cons
         }
     }
     tx->set_len();
+
+    auto fork_config = top::chain_upgrade::xtop_chain_fork_config_center::chain_fork_config();
+    if (!m_is_forked) {
+        auto clock_height = m_store->get_blockchain_height(top::sys_contract_beacon_timer_addr);
+        xdbg_rpc("[sendTransaction_method] check blacklist fork point time, clock height: %" PRIu64, clock_height);
+        if (chain_upgrade::xtop_chain_fork_config_center::is_forked(fork_config.blacklist_function_fork_point, clock_height)) m_is_forked = true;
+    }
+
+    if (m_is_forked) {
```

Comment:
after fork, the m_is_forked is useless, right? if so, no need to store the fork status into m_is_forked. such code will be remove after fork works.

Reply:
just for not to need to load the block after reach forkpoint and before next time upgrade to remove such fork point code